### PR TITLE
user create and update feedback

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- ticket redmine: https://edu-redmine.sun-asterisk.vn/issues/76105

## WHAT (optional)
- User create and update feedback when order is complete

## Evidence (Screenshot or Video)
image
image
image

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: src/commons/dtos/feedbackPageOption.dto.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  @IsNumber()
+  userId: number;
+
+  @IsOptional()
+  @IsNumber()
+  productId: number;
+
+  @IsOptional()
+  @IsBoolean()
+  haveImage: boolean;
```
Comment (huongnt-2545) [inline src/commons/dtos/feedbackPageOption.dto.ts]: thêm giá trị mặc định

File: src/commons/dtos/feedbackPageOption.dto.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  @IsOptional()
+  @IsBoolean()
+  haveContent: boolean;
```
Comment (huongnt-2545) [inline src/commons/dtos/feedbackPageOption.dto.ts]: default value

**Comment by ghost** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**